mapstory/static/style/site/base.less

```diff
@@ -364,4 +364,12 @@ form {
 // same, for {% avatar user 30 %} used in _message_snippet.html
 .inbox-avatar img {
   max-height: 30px;
-}
\ No newline at end of file
+}
+
+.classification_banner_top_padding {
```

Comment:
only other thing is that these would go [here](https://github.com/MapStory/mapstory/blob/master/mapstory/static/style/site/navbar-footer.less) if we're maintaining organization in the less

Reply:
I figured that wasn't the best place to put those.
Why's it called navbar-footer? Isn't that all header stuff?